index.js

```diff
@@ -1,9 +1,15 @@
 'use strict';
 var got = require('got');
-var registryUrl = require('registry-url');
+var npmKeywordUrl = require('npm-keyword-url');
 
-module.exports = function (keyword, cb) {
-	got(module.exports.url(keyword), function (err, data) {
+module.exports = function (keyword, cb, options) {
+	options = options || {};
+	if (typeof options.description === 'undefined') {
+		options.description = true;
+	}
+	var url = npmKeywordUrl(keyword, options.description ? true : false);
```

Comment:
yup. this was left over from a different version i had where i was passing in 2 or 3.